src/Windowing/Silk.NET.Windowing.Common/Interfaces/IView.cs

```diff
@@ -12,7 +12,7 @@ namespace Silk.NET.Windowing
     /// <summary>
     /// Represents a window view.
     /// </summary>
-    public interface IView : IViewProperties, IGLContextSource, IVkSurfaceSource, IDisposable
+    public interface IView : IViewProperties, IGLContextSource, IVkSurfaceSource, IDisposable, INativeWindowSource

```

Comment:
Technically breaking, but I guess this should be fine.
Also note that if the approach I suggested above is adopted, this isn't breaking anymore.

Reply:
Should be fine.